Tutorials/Multilanguage-Setup/index.md

```diff
@@ -4,6 +4,6 @@ meta.Title: "Multilanguage setup in Umbraco"
 product: "CMS"
 ---
 
-# [Language Variants](../../Getting-Started/Backoffice/Variants)
+# Language Variants
 
-In Umbraco 8 you can use **language variants** to setup a multilanguage site.
+In Umbraco 8 you can use **language variants** to setup a multilanguage site. You can read about how to set up and work with language variants [here](../../Getting-Started/Backoffice/Variants).
```

Comment:
I prefer to avoid link texts that are very generic like `here`, so I tried to some up with a rewrite .. What do you think? 🤔 

```suggestion
In Umbraco 8 you can use **language variants** to setup a multilanguage site. You can read about how to set it up and work with a multilingual site in the [language variants article](../../Getting-Started/Backoffice/Variants).
```